compiler/luci/pass/src/ResolveCustomOpMaxPoolWithArgmaxPass.cpp

```diff
@@ -456,7 +469,19 @@ luci::CircleNode *window_y_coord(const std::string &name, float filter_width,
   {
     init_name_and_origin(div, name + "/Div", origin);
 
-    auto divider = create_scalar<loco::DataType::FLOAT32>(graph, 1.0f / filter_width);
+    // Adjustment_coeff is needed to fix computation of quantized tensors
+    //
+    // For example Fp32 value 2.0 could be quantized to 1.996
```

Comment:
(maybe in another PR) `Fp32` and `fp32` are mixed. It would be better to use one convention.